clean_project/search_engine.py

```diff
@@ -38,28 +32,13 @@ def search(self, search_word):
                 connection.close()
         return result_search
 
-    def print_output(self, search_word, results):
-        """
-        Print output of search
-        :param search_word: Word given to search
-        :param results: list of tuples(filename, occurrence in a given file)
-        """
-        if results:
-            print "The word {bold} {word} {end} is contained in the following file/lines:" \
-                .format(bold=BOLD, word=search_word, end=END)
-            for word, filename, occurrence in results:
-                print "--> %s: %s" % (filename, str(occurrence).strip("[]"))
-        else:
-            print "No matches"
-
-
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('word')
     parser.add_argument('connect_path')
     args = parser.parse_args()
     try:
-        results = Search(args.connect_path).search(args.word)
-        Search(args.connect_path).print_output(args.word, results)
+        results = SearchEngine(args.connect_path).search(args.word)
+        search_cli.Search(args.connect_path).print_output(args.word, results)
```

Comment:
really? do i really have to create a searchCli object in order to print the output?